eng/deploy-sql.ps1

```diff
@@ -50,14 +60,7 @@ if ($Username) {
 else {
     $connectionString += "Integrated Security=SSPI;"
 }
-if ($EncryptConnection) {
-    $connectionString += "Encrypt=true;"
-}
-if ($TrustCertificate) {
-    $connectionString += "TrustServerCertificate=true;"
-}
 
-Write-Output "Deploying EPP Evaluation Tracker Application database scripts"
```

Comment:
I re-arranged the code above so that I could print out the connection string on line 50, without including username/password.